blog/_posts/2018-04-18-scala-3.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog-detail
+post-type: blog
+by: Martin Odersky
+title: "Towards Scala 3"
+---
+
+Now that Scala 2.13 is around the corner, it's time to consider the
+roadmap beyond it. It's been no secret that the work on Dotty over the
+last 5 years was intended to explore what a new Scala could look
+like. We are now at a stage where we can commit: Dotty will become
+Scala 3.0.
+
+Of course, this statement invites many follow-up questions. Here are
+some answers we can already give today. We expect there will be more
+questions and answers as things shape up.
+
+_When will it come out?_
+
+The intent is to publish the final Scala 3.0 soon after Scala 2.14. At the
+current release schedule (which might still change), that means late
+2019 or early 2020.
+
+_What is Scala 2.14 for?_
+
+Scala 2.14's main focus will be on smoothing the migration to Scala
+3. It will do this by defining migration tools, shim libraries, and
+targeted deprecations, among others.
+
+_What's new in Scala 3?_
+
+Scala has pioneered the fusion of object-oriented and functional
+programming in a typed setting. Scala 3 will be a big step towards
```

Comment:
for a general audience, better "statically typed" than just "typed", IMO. people coming from dynamic languages don't necessarily consider their languages not to be "typed".

Reply:
I prefer "typed". It's a matter to take back terminology to what it should mean.